features/aanschrijfwijze.feature

```diff
@@ -147,72 +170,73 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+      | O        | Franklin  | van den     | Aedel         | M               | F. van den Aedel           | er is geen adellijke titel vorm die hoort bij geslacht O                                       |
+      | O        | Franklin  | van den     | Aedel         | JV              | F. van den Aedel           | er is geen predicaat vorm die hoort bij geslacht O                                             |
 
     Abstract Scenario: Persoon zonder adellijke titel/predicaat, met een actuele/ontbonden partner met adellijke titel/predicaat heeft aanduiding naamgebruik "E" (eigen naam)
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                        | waarde     |
       | burgerservicenummer         | 999992934  |
       | geslachtsaanduiding (04.10) | <geslacht> |
-      En de persoon heeft de volgende naam gegevens
+      En de persoon heeft de volgende 'naam' gegevens
       | naam                           | waarde      |
       | geslachtsnaam (02.40)          | Groenen     |
       | voornamen (02.10)              | <voornamen> |
       | aanduiding naamgebruik (61.10) | E           |
-      En de persoon heeft een <partner type> partner met <adellijke titel of predicaat type>
+      En de persoon heeft een '<partner type>partner' met de volgende 'naam' gegevens
+      | naam                                 | waarde                              |
+      | adellijke titel of predicaat (02.20) | <adellijke titel of predicaat type> |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999992934                       |
       | fields              | naam.aanschrijfwijze.naam       |
-      Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
       | naam                 | waarde     |
       | aanschrijfwijze.naam | F. Groenen |
 
       Voorbeelden:
       | geslacht | voornamen | partner type | adellijke titel of predicaat type |
-      | M        | Franklin  | actuele      | adellijke titel                   |
-      | V        | Francisca | ontbonden    | predicaat                         |
-      | O        | Franklin  | actuele      | predicaat                         |
-      | O        | Francisca | ontbonden    | adellijke titel                   |
+      | M        | Franklin  |              | B                                 |
+      | V        | Francisca | ex-          | JV                                |
+      | O        | Franklin  |              | JH                                |
+      | O        | Francisca | ex-          | G                                 |
```

Comment:
Hier loop je vooruit op de wens op de ex-partner als type de modelleren. Even in de gaten houden of dat ook de uitkomst is van die discussie. Deze is gestart in https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/1050#issuecomment-1124606005  

Als dit type niet wordt geintroduceerd, dan moet je hier als voorwaarde de aanwezigheid van partner.ontbindingHuwelijkPartnerschap.datum opnemen denk ik.